app/src/main/java/org/oppia/app/profile/AdminSettingsDialogFragment.kt

```diff
@@ -0,0 +1,31 @@
+package org.oppia.app.profile
+
+import android.app.Dialog
+import android.content.Context
+import android.os.Bundle
+import org.oppia.app.fragment.InjectableDialogFragment
+import javax.inject.Inject
+
+const val KEY_ADMIN_SETTINGS_PIN = "ADMIN_SETTINGS_PIN"
+
+class AdminSettingsDialogFragment : InjectableDialogFragment() {
```

Comment:
Here & elsewhere: please add KDocs for new classes.

Reply:
Added KDocs for all new classes.